webpack.config.js

```diff
@@ -75,25 +76,91 @@ const commonConfig = merge([
       rules: [
         {
           test: /\.js$/,
-          // We need to transpile Polymer,so whitelist packages containing ES modules
-          exclude: /node_modules\/(?!(@webcomponents\/shadycss|lit-html|@polymer|@vaadin)\/).*/,
           use: [
-            {
-              loader: 'babel-loader',
-              options: {
-                babelrc: true,
-                extends: join(__dirname + '/.babelrc'),
-                cacheDirectory: true,
-                envName: ENV
-              }
-            },
-            {
-              loader: 'uglify-template-string-loader'
-            }
+            BabelMultiTargetPlugin.loader(),
+            'uglify-template-string-loader'
           ]
         }
       ]
-    }
+    },
+    plugins: [
+      // Babel configuration for multiple output bundles targeting different sets
+      // of browsers
+      new BabelMultiTargetPlugin({
+        babel: {
+          plugins: [
+            [
+              require('@babel/plugin-external-helpers'),
+              {
+                whitelist: helperWhitelist
+              }
+            ],
+
+            [
+              'module:fast-async',
+              {
+                spec: true
+              }
+            ],
+
+            // Minify HTML and CSS in tagged template literals
+            [
+              require('babel-plugin-template-html-minifier'),
+              {
+                modules: {
+                  '@polymer/polymer/lib/utils/html-tag.js': ['html']
+                },
+                htmlMinifier: {
+                  collapseWhitespace: true,
+                  minifyCSS: true,
+                  removeComments: true
+                }
+              }
+            ]
+          ],
+
+          // @babel/preset-env options common for all bundles
+          presetOptions: {
+            // debug: true, // uncomment to debug the babel configuration
+            exclude: ['transform-async-to-generator', 'transform-regenerator'],
+
+            // Don’t add polyfills, they are provided from webcomponents-loader.js
+            useBuiltIns: false
+          }
+        },
+
+        // Modules excluded from targeting into different bundles
+        doNotTarget: [
+          // Array of RegExp patterns
+        ],
+
+        // Modules that should not be transpiled
+        exclude: [
+          // Array of RegExp patterns
+        ],
+
+        // Target browsers with and without ES modules support
+        targets: {
+          es6: {
+            browsers: [
```

Comment:
`browsers` [is ignored](https://babeljs.io/docs/en/babel-preset-env#targetsesmodules) with `esModule: true`. Might want to remove to avoid confusion.